### DISCUSSION TITLE
Styling a child according to the hover state of a parent's sibling

### DISCUSSION THREAD

**Discussion by LeRoiLambda** [tailwindlabs/tailwindcss#12877] [Help] [answered]
Hello!

```javascript
<Link>
    <Image className="hover:scale-105 duration-300" />
</Link>
<div className="relative bg-white ml-12 bottom-10">
    <Link className="hover:text-primaryL">{category}</Link>
    <Link className="group">
        <h1 className="ring-offset-2 duration-300 decoration-transparent underline group-hover:decoration-secondary peer-hover:decoration-secondary">{title}</h1>
        {description && <p>{description}</p>}
        <div>
            <div>
                <div>{author}</div>
                <div>{authorRole}</div>
            </div>
            <Arrow className="group-hover:translate-x-1.5 group-hover:fill-primary duration-300" />
        </div>
    </Link>
</div>
```

If the first link is hover, then, the image should be scaled, the 'decoration-secondary' of the h1 should be "activated", the Arrow should be translated of 1.5 etc..
If the third link(the one with the slug) is hover, same

Does someone know how to do that with tailwind ?

I thank you for your time

**Comment by wongjn** [comment]
Assuming `<Link>` is from `next/link` and `<Image>` is from `next/image`, you could use a parent `group` with `group-has-[a:hover]:` variant like: https://play.tailwindcss.com/9k8fAAzSkZ

**Reply by LeRoiLambda** [reply]
Thanks @wongjn But its not exactly what I'm looking for. In your example, when the second `<a>` is hovered, it activates the other group members. This is exactly what I do not want. Even tho if we just write  `<a class="hover:text-primaryL">{category}</a>`.

What I want: if we hover the last Link, it should **also** trigger the scale of the image.
If we hover the Image(or first Link), it should trigger the translation, outlet etc of the second part

**Reply by wongjn** [reply, marked as answer]
Oh I see, I misinterpreted your request. Try this: https://play.tailwindcss.com/eJyYqj7pVP

**Reply by LeRoiLambda** [reply]
It looks pretty advanced. Thanks, @wongjn. I've never thought about using two groups of hover, one to describe the section being hovered over and the other group to describe the sibling section. It's very powerful!
